hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/observer/ObservableUtil.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.impl.observer;
+
+import com.hazelcast.jet.Observable;
+import com.hazelcast.jet.core.ProcessorMetaSupplier;
+
+import javax.annotation.Nonnull;
+
+import static com.hazelcast.jet.impl.JobRepository.INTERNAL_JET_OBJECTS_PREFIX;
+
+public final class ObservableUtil {
+
+    /**
+     * Constant ID to be used as a {@link ProcessorMetaSupplier#getTags()
+     * PMS tag key} for specifying when a PMS owns an {@link Observable} (ie.
+     * is the entity populating the {@link Observable} with data).
+     */
+    public static final String OWNED_OBSERVABLE = "owned_observable";
+
+    /**
+     * Prefix of all topic names used to back {@link Observable} implementations,
+     * necessary so that such topics can't clash with regular topics used
+     * for other purposes.
+     */
+    private static final String JET_OBSERVABLE_NAME_PREFIX = INTERNAL_JET_OBJECTS_PREFIX + "observable.";
```

Comment:
`observables`

Reply:
Do you mean the constant's name or the value? I've looked through other usages of `INTERNAL_JET_OBJECTS_PREFIX`, but they aren't really consistent enough for me to decide...

Reply:
I meant the prefix should be `observables.`, also this should be moved to JobRepository where all the other prefixes are